drivers/volume/volume.go

```diff
@@ -83,13 +83,6 @@ var (
 		CSIDriverName,
 		KDMPDriverName,
 	}
-	csiDriverWithoutSnapshotSupport = []string{
-		vSphereCSIProvisioner,
```

Comment:
These constants needs to be removed from constants also.. static check is failing